documentation/source/overview/introduction.rst

```diff
@@ -4,15 +4,32 @@ Introduction
 Context
 -------
 
-For the past 25 years, the field of neuroimaging has witnessed the development of several software packages for processing multi-parametric magnetic resonance imaging (mpMRI) to study the brain. These software packages are now routinely used by researchers and clinicians, and have contributed to important breakthroughs for the understanding of brain anatomy and function.
+For the past 25 years, the field of neuroimaging has witnessed the development of several software
+packages for processing multi-parametric magnetic resonance imaging (mpMRI) to study the brain.
+These software packages are now routinely used by researchers and clinicians, and have contributed
+to important breakthroughs for the understanding of brain anatomy and function.
```

Comment:
It looks like the line break changes from #3197 were accidentally undone. This file should be unchanged, I think.

Reply:
Hmm you're right I think. What was the logic for removing the line breaks?